app/src/screens/PersonCredential.tsx

```diff
@@ -139,6 +143,14 @@ export default function PersonCredential() {
     // and the user needs to wait.
     setWorkflowInProgress(true)
 
+    timer.current = setTimeout(() => {
+      if (!didCompleteAttestationProofRequest) {
```

Comment:
Should this be checking if the proof request was completed or if it was recieved?